### ISSUE TITLE
Resync Issue

### ISSUE THREAD

**Issue by DereckR** [tidusjar/Ombi#1173]
#### Ombi Version:

V 2.1.1436

#### Update Branch:

Early Access Preview

#### Media Sever:

Plex

#### Media Server Version:

1.4.0.3224

#### Operating System:

Windows 10 x64

#### Applicable Logs (from `/logs/` directory or the Admin page):

```

(Logs go here. Don't remove the ``` tags for showing your logs correctly. Please make sure you remove any personal information from the logs)

```

#### Problem Description:

I had an issue where apparently Ombi was not sending requests to Sonarr and Couch at some point. However I have switched to Radarr now. Is there a way to force push all unavailable tv shows and movies to resync?

Thanks,
Dereck

#### Reproduction Steps:

None to reproduce.

**Comment by SuperPotatoMen** [comment]
No that isn't possible you need to manualy delete and request it again :(

**Comment by DereckR** [comment]
When I delete and try to request it again says it has already been requested....

**Comment by DereckR** [comment]
Just tested and does this on movies but lets me request it again on TV shows.

**Comment by SuperPotatoMen** [comment]
I'm not

**Comment by SuperPotatoMen** [comment]
I'm not following your question. You like to resend some request to radarr instead of cp right?

**Comment by DereckR** [comment]
@SuperPotatoMen What happen was apparently at some point Ombi stopped sending requests to Sonarr, CouchPotato. I am noticing that some TV shows and Movies never made it to either one. So I noticed this when I was installing Radarr to begin testing it.

So what I am trying to do is push all of the requests from Ombi to sync with all of these services to make sure they all made it. With this being said now that I am using Radarr there is nothing requested on it because the items were request prior to install.

I had followed the steps by removing the movie then trying to re add it but to find out that it won't let me request it again shows already has been.
